packages/expansion-hub/src/ExpansionHub.ts

```diff
@@ -47,7 +48,44 @@ export interface ExpansionHub extends RevHub {
 
     // Device Control
     getBulkInputData(): Promise<BulkInputData>;
-    getADC(): Promise<number>;
+
+    /**
+     * Read the value of an analog channel in mV.
+     *
+     * @param channel
+     */
+    getAnalogInput(channel: number): Promise<number>;
+
+    /**
+     * For SystemType.DigitalIO, the result is the total current running through
+     * the shared ground of the digital channels.
+     *
+     * @param systemType the type of system to get current for. Defaults to Battery, which is the whole device's current.
+     */
+    getSystemCurrent(systemType?: SystemCurrentType): Promise<number>;
+
+    /**
+     * Get the current draw of a given motor.
+     *
+     * @param motorChannel
+     */
+    getMotorCurrent(motorChannel: number): Promise<number>;
+
+    /**
+     * get the battery's voltage (mV)
+     */
+    getBatteryVoltage(): Promise<number>;
+
+    /**
+     * Check the 5V line's voltage (mV)
+     */
+    get5VMonitorVoltage(): Promise<number>;
```

Comment:
The purpose of calling this method is to use it to adjust the output of `getAnalogInput()`. 
Rather than exporting this method, we should just do what it does within `getAnalogInput()` to make the values it outputs more accurate. Talk to the electrical team for details on the simple math involved.

Reply:
Fun fact: The FTC SDK doesn't currently do this. We should fix that as well (for now I'm leaving a TODO comment uncommitted in my IDE).

Reply:
After talking to the electrical engineers, it appears that this method is not intended to adjust the output of getAnalogInput. It measures the voltage on the 5V bus, which is useful if you are using a sensor where the output is dependent on VCC.